app/models/agents/website_agent.rb

```diff
@@ -259,13 +261,24 @@ def check_urls(in_url)
       end
     end
 
+    def get_valid_url(url)
+      if (url =~ URI::ABS_URI) 
+      	return url
```

Comment:
Why do we not want to consider them if they match `URI::ABS_URI`.  Because then they must already be valid? 

Does this work with relative URLs?